### PR TITLE
Update CDH version to 5.15.1

### PR DESCRIPTION
**Pull request by MiguelWeezardo**

### CODE REVIEW

File: etc/compose/cdh5.15-hive/docker-compose.yml [hunk 1]
```diff
@@ -2,5 +2,5 @@ version: '2.0'
 services:
```
Comment (kokosing) [inline etc/compose/cdh5.15-hive/docker-compose.yml]: I would not rename, but copy. So we could have two versions for a while.

Comment (findepi) [reply etc/compose/cdh5.15-hive/docker-compose.yml]: I would rename, not copy. I don't see a need for 2 versions (also, cdh5.13:6 won't disappear, we can use that for as long as we want)

**Comment by kokosing** [comment]
> Can you also add CDH 5.12

Also I think I can build such if needed. So nevermind this comment.

**Comment by kokosing** [comment]
Merged, thanks!

**Pull request merged**